src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -490,6 +490,14 @@ def assemble_server_py(parsed_question, location):
     except:
         raise
 
+    server_py = re.sub(
+        r"(data2?\[(?P<Quote1>\"|\')params(?P=Quote1)\]\[(?P<Quote2>\"|\')part\d+(?P=Quote2)\]"
+        + r"\[(?P<Quote3>\"|\')ans\d+(?P=Quote3)\]\[(?P<Quote4>\"|\')value(?P=Quote4)\] ?= "
+        + r"?f?(?P<Opening>\"{3}|\'{3}|\"|\'|).*?(?P=Opening)$)",
+        lambda match: backticks_to_code_tags(match.expand(r"\1")),
+        server_py,
+        flags=re.MULTILINE | re.DOTALL,
+    )
```

Comment:
This might be the wrong spot to do it, or we might want to put something like this under a flag, so it doesn't effect the public and instructor markdown generation?